refarch-frontend/vite.config.ts

```diff
@@ -0,0 +1,41 @@
+// Plugins
+import { fileURLToPath, URL } from "node:url";
+
+import vue from "@vitejs/plugin-vue";
+import ViteFonts from "unplugin-fonts/vite";
+// Utilities
+import { defineConfig } from "vite";
+import vuetify, { transformAssetUrls } from "vite-plugin-vuetify";
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [
+    vue({
+      template: { transformAssetUrls },
+    }),
+    // https://github.com/vuetifyjs/vuetify-loader/tree/next/packages/vite-plugin
+    vuetify({
+      autoImport: true,
+    }),
+    ViteFonts({
+      google: {
+        families: [
+          {
+            name: "Roboto",
+            styles: "wght@100;300;400;500;700;900",
+          },
+        ],
+      },
+    }),
```

Comment:
I wonder that we explicitly "import" Roboto font here. Where are the font getting pulled from? Are they included in the final build or loaded externally from Google servers? Some browsers might have them integrated out of the box, but others don't. We shouldn't force loading from external servers and make sure fonts are always included. On top of that, why do we need to define font here? Isn't this Vuetify default?